### PR TITLE
itree relative paths fix, imeta updates, and cppzmq deprecation fix (main)

### PR DESCRIPTION
**Pull request by korydraughn**
TODO: Add test to irods/irods test suite.

### CODE REVIEW

**Comment by korydraughn** [comment]
test available at https://github.com/irods/irods/pull/6156

File: src/imeta.cpp [hunk 1]
```diff
@@ -1214,6 +1226,7 @@ parse_program_options(int _argc,
         ("help,h",                                             "Show imeta help")
         ("verbose,v",                                          "Verbose output")
         ("very_verbose,V",                                     "Very verbose output")
+        ("as_admin,M",                                         "Execute as administrator")
```
Comment (alanking) [inline src/imeta.cpp:1229]: Would it be better to use "admin_mode" instead of "as_admin" for consistency?

File: src/imeta.cpp [hunk 2]
```diff
@@ -2208,6 +2227,10 @@ void usageMain() {
         "The appropriate zone (local or remote) is determined from the path names",
         "or via -z Zonename (for 'qu' and when working with resources).",
         " ",
+        "If you are the iRODS administrator, you can include the -M option to",
```
Comment (alanking) [inline src/imeta.cpp:2245]: "the iRODS administrator" -> "an iRODS administrator" ?

**Review by alanking — COMMENTED** [review]
Couple of small suggestions. Otherwise looks good

File: src/imeta.cpp (hunk 1 above)
Comment (korydraughn) [reply src/imeta.cpp:1229]: sure. will change it.

File: src/imeta.cpp (hunk 2 above)
Comment (korydraughn) [reply src/imeta.cpp:2245]: good catch. will change it.

**Review by alanking — APPROVED** [review]

File: src/imeta.cpp (hunk 1 above)
Comment (korydraughn) [reply src/imeta.cpp]: done

File: src/imeta.cpp (hunk 2 above)
Comment (korydraughn) [reply src/imeta.cpp]: done

**Comment by korydraughn** [comment]
added #'s

tests included in https://github.com/irods/irods/pull/6156

**Comment by trel** [comment]
last #'d issue seems... the wrong one?

**Comment by trel** [comment]
oh 6173 and 6174 got flipped

**Comment by korydraughn** [comment]
dang. fixing ...

**Comment by korydraughn** [comment]
fixed it.

**Pull request merged**